src/gui/qgssymbolbutton.h

```diff
@@ -268,6 +275,13 @@ class GUI_EXPORT QgsSymbolButton : public QToolButton
      */
     void setToDefaultSymbol();
 
+    /**
+     * Insures that the symbol height is always short.
+     * 
+     * \since QGIS 4.0
+     */
+    void setForceShortSize( bool shortSize );
```

Comment:
What's this actually MEAN?

Reply:
What does life really mean @nyalldawson ?! :wink: the name was quite bad, I agree. What this is for is to allow for marker symbol buttons to be as short as their polygon and line counterparts (i..e not the tail block-ish size that was added to the button a few versions ago). 

Better naming would be? setOGSize? ;-P

Reply:
Renamed to fixedSizeConstraints() / setFixedSizeConstraints(bool)